profile-widget/profilescene.cpp

```diff
@@ -350,8 +366,14 @@ void ProfileScene::plotDive(const struct dive *dIn, int dcIn, DivePlannerPointsM
 	create_plot_info_new(d, get_dive_dc_const(d, dc), &plotInfo, !calcMax, planner_ds);
 
 	bool hasHeartBeat = plotInfo.maxhr;
-	updateVisibility(hasHeartBeat);
-	updateAxes(hasHeartBeat);
+	// For mobile we might want to turn of some features that are normally shown.
+#ifdef SUBSURFACE_MOBILE
+	bool simplified = true;
```

Comment:
... except that this hardcodes it. Which is fine for now, of course, but should then turn into an option on mobile that takes screen space into account...